File: proyecto6/circulo.php
<?php

class Circulo {
    public $radio;
    public $pi = 3.14;
    public function calcularArea() {
        return 'Area = '.$this->pi * $this->radio * $this->radio;
    }
}

$area = new Circulo;
$area -> radio = 5;
$area -> pi;
echo $area -> calcularArea();
?>

File: Java7/Ejercicio_82/paqueteArea/Circulo.java
package paqueteArea;
import java.lang.Math;

public class Circulo extends Figura implements Calculable{
	
	double radio;
	
	public Circulo (String color, double radio) {
		super(color);
		this.radio = radio;
	}
	
	@Override
	public double calcularArea() {
		return Math.PI * radio * radio;
	}
}
